packages/compass-indexes/src/components/search-indexes-modals/update-search-index-modal.tsx

```diff
@@ -59,6 +66,7 @@ const mapState = ({
     null,
     2
   ),
+  indexType: indexes.find((x) => x.name === indexName)?.type,
```

Comment:
Should we do the `find` only once instead of two times, here and on the line above?